### PR TITLE
Updated sections based on meeting notes

### PR DESCRIPTION
**Pull request by erikmwilson**
Updates Overview, Getting Started, and Using sections.

### CODE REVIEW

**Comment by erikmwilson** [comment]
@robb-romans @kmsholcomb Take a look and let me know if there are any changes.

File: user-guide/doc/using/index.rst [hunk 1]
```diff
@@ -6,8 +6,11 @@ Using
 
 .. Define |product name| in conf.py
 
-Include tasks that can be completed using the product.
+Include tasks that can be completed using the product. Create as many task as
```
Comment (robb-romans) [inline user-guide/doc/using/index.rst]: create as many tasks

File: user-guide/doc/using/task-template.rst [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 
-======
-Task 3
-======
+=============
+Task template
+=============
 
-Provide any concepts or context necessary to understand the task.
+Name tasks with a gerund (for example, Adding a user).
```
Comment (robb-romans) [inline user-guide/doc/using/task-template.rst]: @kmsholcomb do we want gerunds for these?

File: user-guide/doc/using/use-cases/index.rst [hunk 3]
```diff
@@ -6,7 +6,10 @@ Uses cases
 
 .. Define |product name| in conf.py
 
-Include use cases that can be completed using the product.
+Include use cases that can be completed using the product. Use-cases are
```
Comment (robb-romans) [inline user-guide/doc/using/use-cases/index.rst]: Use cases (without hyphen) (applies throughout)

File: user-guide/doc/using/use-cases/use-case-template.rst [hunk 4]
```diff
@@ -0,0 +1,17 @@
+.. _use-case-1-xxx-ug:
+
+==========
+Use case 1
+==========
+
+Desribe the use-case. Link to any tasks required to complete the use-case.
```
Comment (robb-romans) [inline user-guide/doc/using/use-cases/use-case-template.rst]: Describe

File: user-guide/doc/using/use-cases/use-case-template.rst [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+You can use the task-template to create a use-case process. Each step in the
```
Comment (robb-romans) [inline user-guide/doc/using/use-cases/use-case-template.rst]: task template

File: user-guide/doc/using/task-template.rst (hunk 2 above)
Comment (kmsholcomb) [inline user-guide/doc/using/task-template.rst]: For task titles, we should just use the imperative: Add a user.

File: user-guide/doc/using/index.rst (hunk 1 above)
Comment (erikmwilson) [inline user-guide/doc/using/index.rst]: Done.

File: user-guide/doc/using/use-cases/index.rst (hunk 3 above)
Comment (erikmwilson) [inline user-guide/doc/using/use-cases/index.rst]: Done.

File: user-guide/doc/using/use-cases/use-case-template.rst (hunk 4 above)
Comment (erikmwilson) [inline user-guide/doc/using/use-cases/use-case-template.rst]: Done.

File: user-guide/doc/using/use-cases/use-case-template.rst (hunk 5 above)
Comment (erikmwilson) [inline user-guide/doc/using/use-cases/use-case-template.rst]: Done.

**Comment by erikmwilson** [comment]
@robb-romans @kmsholcomb This should be ready to merge now.

**Comment by kmsholcomb** [comment]
@erikmwilson @robb-romans: Sorry, I just started my review. I'll be as quick as I can.

File: user-guide/doc/getting-started/navigating.rst [hunk 6]
```diff
@@ -0,0 +1,10 @@
+.. _navigating-xxx-ug:
+
+==========
+Navigating
+==========
+
+Provide information about how to navigate the product
+(for example, description of UI, menus).
```
Comment (kmsholcomb) [inline user-guide/doc/getting-started/navigating.rst]: (for example, description of the UI and menus).

File: user-guide/doc/getting-started/navigating.rst [hunk 7]
```diff
@@ -0,0 +1,10 @@
...
+Avoid screen shots. If steps are necessary, follow the layout in the
```
Comment (kmsholcomb) [inline user-guide/doc/getting-started/navigating.rst]: Avoid using screenshots.

File: user-guide/doc/using/index.rst [hunk 8]
```diff
@@ -6,8 +6,11 @@ Using
 
 .. Define |product name| in conf.py
 
-Include tasks that can be completed using the product.
+Include tasks that can be completed using the product. Create as many tasks as
+necessary using the task template. Tasks can be organized by categories.
 
+Tasks in the Using section are generally end-user tasks. Tasks completed
+by an administrator (maintenance tasks) go in the Administering section.
```
Comment (kmsholcomb) [inline user-guide/doc/using/index.rst]: ... by an administrator, or maintenance tasks, go in the ...

File: user-guide/doc/using/task-template.rst [hunk 9]
```diff
@@ -1,13 +1,13 @@
...
 
-======
-Task 3
-======
+=============
+Task template
+=============
 
-Provide any concepts or context necessary to understand the task.
+Name tasks with the imperative (for example, Add a user).
```
Comment (kmsholcomb) [inline user-guide/doc/using/task-template.rst]: Begin the title of tasks with an imperative verb (for example, Add a user).

File: user-guide/doc/using/task-template.rst [hunk 10]
```diff
@@ -21,31 +21,13 @@ To access |product name|
...
-
    Result of step 2.
 
-Provide the results of completing the task.
+Result of completing the task.
 
 Next steps
 ~~~~~~~~~~
 
 Provide links to any tasks that must/can be completed after this task.
```
Comment (kmsholcomb) [inline user-guide/doc/using/task-template.rst]: Provide links to any tasks that must or can be completed ....

File: user-guide/doc/using/use-cases/index.rst [hunk 11]
```diff
@@ -6,7 +6,10 @@ Uses cases
 
 .. Define |product name| in conf.py
 
-Include use cases that can be completed using the product.
+Include use cases that can be completed using the product. use cases are
```
Comment (kmsholcomb) [inline user-guide/doc/using/use-cases/index.rst]: Describe the high-level use cases (scenarios, goals) that user can accomplish by using the product. Use cases are ...

File: user-guide/doc/using/use-cases/index.rst [hunk 12]
```diff
@@ -6,7 +6,10 @@ Uses cases
...
+generally large processes made up of multiple tasks. Create as many use cases
+as necessary using the use case template.
+use cases can be organized by categories.
```
Comment (kmsholcomb) [inline user-guide/doc/using/use-cases/index.rst]: Use cases ...

File: user-guide/doc/getting-started/accessing.rst [hunk 13]
```diff
@@ -4,48 +4,7 @@
...
-Next steps
-~~~~~~~~~~
-
-Provide links (to any task that must be completed after this task.
-
-:ref:`task-2-xxx-ug`.
+Provide information about how to access the product. This could include links
+to the product, how to log in, how to authenticate.
+If steps are necessary, follow the layout in the
+:ref:`Task template<task-template-xxx-ug>`.
```
Comment (erikmwilson) [inline user-guide/doc/getting-started/accessing.rst]: Done.

File: user-guide/doc/getting-started/index.rst [hunk 14]
```diff
@@ -7,18 +7,7 @@ Getting started
...
-  request access
-
-- Navigating - overview of the UI (if any)
-
-- Configuring - configuration task to get the product to a usable state
-
-- Authenticating - how to authenticate the user or product
-
-Add all getting stated task filenames to the TOC list.
+Some examples are included below but you can add (or remove) topics.
```
Comment (erikmwilson) [inline user-guide/doc/getting-started/index.rst]: Done.

File: user-guide/doc/getting-started/navigating.rst (hunk 6 above)
Comment (erikmwilson) [inline user-guide/doc/getting-started/navigating.rst]: Done.

File: user-guide/doc/getting-started/navigating.rst (hunk 7 above)
Comment (erikmwilson) [inline user-guide/doc/getting-started/navigating.rst]: Done.

File: user-guide/doc/using/index.rst (hunk 8 above)
Comment (erikmwilson) [inline user-guide/doc/using/index.rst]: Done.

File: user-guide/doc/using/task-template.rst (hunk 9 above)
Comment (erikmwilson) [inline user-guide/doc/using/task-template.rst]: Done.

File: user-guide/doc/using/task-template.rst (hunk 10 above)
Comment (erikmwilson) [inline user-guide/doc/using/task-template.rst]: Done.

File: user-guide/doc/using/use-cases/index.rst (hunk 11 above)
Comment (erikmwilson) [inline user-guide/doc/using/use-cases/index.rst]: Done.

File: user-guide/doc/using/use-cases/index.rst (hunk 12 above)
Comment (erikmwilson) [inline user-guide/doc/using/use-cases/index.rst]: Done.

**Comment by erikmwilson** [comment]
@robb-romans @kmsholcomb This should be ready now.

**Pull request merged**